android/src/main/java/com/oney/WebRTCModule/DailyAudioManager.java

```diff
@@ -19,23 +20,20 @@
 import java.util.Set;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import com.oney.WebRTCModule.WebRTCDevicesManager.DeviceType;
+import com.oney.WebRTCModule.WebRTCDevicesManager.AudioRoute;
 
 public class DailyAudioManager implements AudioManager.OnAudioFocusChangeListener {
     static final String TAG = DailyAudioManager.class.getCanonicalName();
 
     public enum Mode {
         IDLE,
         VIDEO_CALL,
-        VOICE_CALL
-    }
-
-    private enum DeviceType {
-        BLUETOOTH,
-        HEADSET,
-        SPEAKER,
-        EARPIECE
+        VOICE_CALL,
+        USER_DEFINED
```

Comment:
wonder if we should name this so that it says "route" somewhere so it's easy to tell that this means "the user is choosing the audio route"

Reply:
I like It. Will rename to that.